config/dev.env.js

```diff
@@ -3,6 +3,6 @@ const merge = require('webpack-merge')
 const prodEnv = require('./prod.env')
 
 module.exports = merge(prodEnv, {
-    API_URI: '"http://127.0.0.1:8000/graphql"',
+    API_URI: '"http://127.0.0.1:8080/graphql"',
```

Comment:
I don't think this is the correct change. The Federation site consists of two separate applications - the backend and the frontend. The Python backend (which servers the API) runs by default on port 8000. The frontend runs on 8080.

Reply:
Thanks, I saw the backend code today so this makes more sense now.